src/ovp_pipeline/prompt_registry.py

```
@@ -0,0 +1,280 @@
+        if fm_version != expected_version:
+            raise PromptRegistryError(
+                f"{path}: frontmatter version={fm_version!r} but filename "
+                f"implies {expected_version!r}"
+            )
+
+        status = str(metadata["status"]).lower()
+        valid_statuses = {"stable", "experimental", "deprecated"}
+        if status not in valid_statuses:
+            raise PromptRegistryError(
+                f"{path}: frontmatter status={status!r} not in {sorted(valid_statuses)}"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)